schema/configuration_endpoint.json

```diff
@@ -62,9 +66,11 @@
         }
       }
     },
-    "nonce": {
+    "generation": {
```

Comment:
How about "timestamp" which is less ambiguous and consistent to other format=date-time fields?